modules/flowable-spring-boot/flowable-spring-boot-starters/flowable-spring-boot-autoconfigure/src/test/java/org/flowable/test/spring/boot/AllEnginesAutoConfigurationTest.java

```diff
@@ -171,7 +170,6 @@ public void usingAllAutoConfigurationsTogetherShouldWorkCorrectly() {
             SpringDmnEngineConfigurator dmnConfigurator = context.getBean(SpringDmnEngineConfigurator.class);
             SpringFormEngineConfigurator formConfigurator = context.getBean(SpringFormEngineConfigurator.class);
             SpringIdmEngineConfigurator idmConfigurator = context.getBean(SpringIdmEngineConfigurator.class);
-            SpringEventRegistryConfigurator eventConfigurator = context.getBean(SpringEventRegistryConfigurator.class);
```

Comment:
Nice spot about this. However, let's not remove it and instead add

```
assertThat(cmmnEngineConfiguration.isDisableEventRegistry()).isTrue();
            assertThat(cmmnEngineConfiguration.getEventRegistryConfigurator()).isNull();
assertThat(processEngineConfiguration.isDisableEventRegistry()).isTrue();
            assertThat(processEngineConfiguration.getEventRegistryConfigurator()).isNull();
assertThat(appEngineConfiguration.getEventRegistryConfigurator())
                    .as("AppEngineConfiguration idmEngineConfigurator")
                    .isSameAs(idmConfigurator);
```

in the appropriate places below. Similar to how it is done for the IDM

Reply:
Took my best shot at what was missing; correct as needed.  Thanks

Reply:
Thanks, it looks as expected. Once the CI is done, I'll merge it

Reply:
thanks